package.json

```diff
@@ -38,13 +38,16 @@
     "url": "git://github.com/wikimedia/jquery.uls.git"
   },
   "bugs": {
-    "url" : "https://github.com/wikimedia/jquery.uls/issues"
+    "url": "https://github.com/wikimedia/jquery.uls/issues"
   },
   "engine": {
     "node": ">=6.11.1"
   },
- "license": "(MIT OR GPL-2.0+)",
- "scripts": {
-  "test": "grunt test --verbose"
+  "license": "(MIT OR GPL-2.0+)",
+  "scripts": {
+    "test": "grunt test --verbose"
+  },
+  "dependencies": {
+    "browserify": "^16.2.2"
```

Comment:
Can you explain this addition? We need browserify to update the language-data but it is abtracted using scripts/fetch-language-data.sh. browserify is not a dependency to jquery.uls.